docs/src/modules/spring/pages/using-acls.adoc

```diff
@@ -0,0 +1,192 @@
+= Using ACLs
+
+include::ROOT:partial$include.adoc[]
+
+Access Control Lists (ACLs) allow you to control which services and sources of requests may access your services.
+
+Within a Kalix project, all communication between services uses Mutual TLS (mTLS). This is injected transparently by Kalix. The clients you use to make calls from one service to another do NOT need to be configured to do this. Kalix transparently captures outgoing requests to other services and wraps them in an mTLS connection.
+
+Based on this mTLS support, Kalix is able to read and apply policies based on where the request was made from.
+
+== Principals
+
+A principal in Kalix is an abstract concept that represents anything that can make or be the source of a request. Principals that are currently supported by Kalix include other services, and the internet. Kalix uses the above described mTLS support to associate requests with one or more principals.
```

Comment:
But then we have `service` and `principal` as settings. So it seems that the category Principal is divided in two. One is called `principal` and the other `service`. It seems it's difficult to know when one talks about the principal to which one is referring.

Reply:
I agree, but that was defined as such in gRPC. At the end, 'service' is also a principal. Confusing, but we will have to go with it for now.